### ISSUE TITLE
RGW not working in octopus/pacific

### ISSUE THREAD

**Issue by tspmelo** [SUSE/sesdev#425]
I just created 2 cluster, octopus and pacific, and both don't have RGW properly configured.
It shows the following when I try to access any RGW page in the dashboard:
```
Information
No RGW credentials found, please consult the documentation on how to enable RGW for the dashboard.
Please consult the documentation on how to configure and enable the Object Gateway management functionality.
```

**Comment by sebastian-philipp** [comment]
you'll need to set up the admin user and then set up the credentials in the dashboard

**Comment by asettle** [comment]
https://docs.ceph.com/docs/master/mgr/dashboard/#enabling-the-object-gateway-management-frontend

**Comment by smithfarm** [comment]
I'm not sure it ever worked? 

https://tracker.ceph.com/issues/44605

**Comment by asettle** [comment]
> I'm not sure it ever worked?
> 
> https://tracker.ceph.com/issues/44605

Correct. Follow the steps I linked above to setup the admin users and credentials.

**Comment by smithfarm** [comment]
> Correct. Follow the steps I linked above to setup the admin users and credentials.

I wonder, though, whether it wouldn't be easier to just wait for the above feature to get merged and backported to Octopus ;-)

**Comment by asettle** [comment]
> > Correct. Follow the steps I linked above to setup the admin users and credentials.
> 
> I wonder, though, whether it wouldn't be easier to just wait for the cephadm feature to get merged and backported to Octopus ;-)

Not if you've got to document a feature that isn't available :+1:

**Comment by smithfarm** [comment]
@asettle Indeed, thanks to your documentation, everyone can now manually activate RGW in the dashboard (even in a sesdev cluster) and work around this cephadm bug today. What I meant to write was: instead of automating this manual procedure in sesdev now, it will be easier to wait for the bug to be fixed in cephadm.
